### PR TITLE
add opportunity for metered/unmetered BillingMethod

### PR DESCRIPTION
**Pull request by andreschab90**
* metered billingMethod will be used as a default value
* this PR should allow to use the unmetered BillingMethod

### CODE REVIEW

File: step.yml [hunk 1]
```diff
@@ -52,6 +52,12 @@ inputs:
       summary: ""
       description: "Project ARNs can be obtained using the AWS CLI 'devicefarm list-projects' command."
       is_required: true
+  - billing_method: ""
```
Comment (fadookie) [inline step.yml]: I think you can change this to `billingMethod: "METERED"` to set the default to metered as this is probably what most people will be using.

File: step.yml [hunk 2]
```diff
@@ -52,6 +52,12 @@ inputs:
...
+    opts:
+      title: "Billing Method"
+      summary: ""
+      description: "use METERED or UNMETERED"
```
Comment (fadookie) [inline step.yml]: Can you make this a little more descriptive? How about `"Specifies the billing method for a test run. Use \`METERED\` for free tier and pay-as-you-go billing, and \`UNMETERED\` to use pre-paid device slots. See [How to purchase device slots](https://docs.aws.amazon.com/devicefarm/latest/developerguide/how-to-purchase-device-slots.html) for more info."`

File: step.yml [hunk 3]
```diff
@@ -52,6 +52,12 @@ inputs:
...
+      is_required: false
```
Comment (fadookie) [inline step.yml]: also please add this code to make this a drop-down list:
```
      value_options:
        - "METERED"
        - "UNMETERED"
```

File: step.sh [hunk 4]
```diff
@@ -283,6 +284,7 @@ fi
 echo_details "* device_farm_project: $device_farm_project"
 echo_details "* test_package_name: $test_package_name"
 echo_details "* test_type: $test_type"
+echo_details "* billing_method: $billing_method"
```
Comment (fadookie) [inline step.sh]: also around line 306 please add the following code to validate the expected option was passed:
```
options=("METERED"  "UNMETERED")
validate_required_input_with_options "billing_method" "${billing_method}" "${options[@]}"
```

File: step.yml (hunk 1 above)
Comment (andreschab90) [inline step.yml]: fixed

File: step.yml (hunk 2 above)
Comment (andreschab90) [inline step.yml]: thanks. updated

File: step.yml (hunk 3 above)
Comment (andreschab90) [inline step.yml]: Done

File: step.sh (hunk 4 above)
Comment (andreschab90) [inline step.sh]: fixed

**Comment by andreschab90** [comment]
@fadookie PR is updated. Thanks war your amazing input :-)

**Comment by fadookie** [comment]
👍 great work

**Pull request merged**